canonical-websites/www.ubuntu.com
# 1592 internet of things contact us

## Done

* update /internet-of-things/contact-us to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/internet-of-things/contact-us)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/11.%20Forms?preview=iot-contact.png) and the [demo](http://www.ubuntu.com-1592-internet-of-things-contact-us.demo.haus/internet-of-things/contact-us)

## Issue / Card

Fixes #1592
